src/config/animations.ts

```diff
@@ -0,0 +1,7 @@
+import { Platform, UIManager } from 'react-native';
+
+if (Platform.OS === 'android') {
+  if (UIManager.setLayoutAnimationEnabledExperimental) {
+    UIManager.setLayoutAnimationEnabledExperimental(true);
```

Comment:
Interesting, never used this one, what does it do?

Reply:
this is my first time using it too. It enables automatic animations in view changes by calling `configureNext` before the change 😮